translations/ja.json

```diff
@@ -547,70 +547,70 @@
+	"Sorry, the IP provided with the --listen-address flag is invalid: {{.listenAddr}}.": "申し訳ありませんが、--listen-address フラグで指定された IP アドレスは無効です: {{.listenAddr}}",
+	"Sorry, the address provided with the --insecure-registry flag is invalid: {{.addr}}. Expected formats are: \u003cip\u003e[:\u003cport\u003e], \u003chostname\u003e[:\u003cport\u003e] or \u003cnetwork\u003e/\u003cnetmask\u003e": "申し訳ありませんが、--insecure-registry で指定されたアドレス {{.addr}} は無効です。想定された形式: \u003cIP\u003e[:\u003cポート\u003e]、\u003cホスト名\u003e[:\u003cポート\u003e]、\u003cネットワーク\u003e/\u003cネットマスク\u003e",
+	"Sorry, the kubeadm.{{.parameter_name}} parameter is currently not supported by --extra-config": "申し訳ありませんが、kubeadm.{{.parameter_name}} パラメーターは現在 --extra-config で未対応です",
+	"Sorry, the url provided with the --registry-mirror flag is invalid: {{.url}}": "申し訳ありませんが、--registry-mirror フラグとともに指定された URL は無効です: {{.url}}",
+	"Sorry, {{.driver}} does not allow mounts to be changed after container creation (previous mount: '{{.old}}', new mount: '{{.new}})'": "申し訳ありませんが、{{.driver}} はコンテナーの生成後にマウントを変更できません (旧マウント: '{{.old}}'、新マウント: '{{.new}})'",
+	"Source {{.path}} can not be empty": "ソース {{.path}} は空にできません。",
+	"Specified Kubernetes version {{.specified}} is less than the oldest supported version: {{.oldest}}": "指定された Kubernetes バージョン {{.specified}} はサポートされたバージョン {{.oldest}} より古いです",
+	"Specify --kubernetes-version in v\u003cmajor\u003e.\u003cminor.\u003cbuild\u003e form. example: 'v1.1.14'": "v\u003cメジャー\u003e.\u003cマイナー.\u003cビルド\u003e形式で --kubernetes-version 値を指定します。例: 'v1.1.14'",
+	"Specify an alternate --host-only-cidr value, such as 172.16.0.1/24": "代わりの --host-only-cidr 値を指定します (172.16.0.1/24 など)",
+	"Specify arbitrary flags to pass to the Docker daemon. (format: key=value)": "Docker デーモンに渡す任意のフラグを指定します。(形式: key=value)",
+	"Specify arbitrary flags to pass to the build. (format: key=value)": "ビルドに渡す任意のフラグを指定します。(形式: key=value)",
+	"Specify the 9p version that the mount should use": "マウントが使用する 9p バージョンを指定します",
+	"Specify the ip that the mount should be setup on": "マウントが使用する IP アドレスを指定します",
+	"Specify the mount filesystem type (supported types: 9p)": "マウントするファイルシステムタイプを指定します (対応したタイプ: 9p)",
+	"Specify the port that the mount should be setup on, where 0 means any free port.": "マウントが使用するポート番号を指定します (0 は任意の空きポートを使用)。",
+	"Specifying extra disks is currently only supported for the following drivers: {{.supported_drivers}}. If you can contribute to add this feature, please create a PR.": "追加ディスク指定は現在 {{.supported_drivers}} ドライバーのみ対応しています。本機能の追加に貢献可能な場合、PR を作成してください。",
+	"StartHost failed, but will try again: {{.error}}": "StartHost に失敗しましたが、再度試してみます: {{.error}}",
+	"Starting control plane node {{.name}} in cluster {{.cluster}}": "コントロールプレーンの {{.name}} ノードを {{.cluster}} クラスター上で起動しています",
+	"Starting minikube without Kubernetes {{.name}} in cluster {{.cluster}}": "{{.cluster}} クラスター中の Kubernetes {{.name}} なしで minikube を起動しています",
```

Comment:
What means this?
minikube starts without kubernetes? :thinking:
Can you share more context for this message for review?

Reply:
See https://minikube.sigs.k8s.io/docs/faq/#can-i-start-minikube-without-kubernetes-running

Reply:
I see! :smile: 
Thank you to quick respose.